e2e_tests/test_browse.py

```diff
@@ -45,25 +55,23 @@ def test_browse_filter_functionality_with_query_string_parameters(
         )
 
         header_rows = aau_user_page.locator(
-            "#tbl_result tr:visible"
-        ).evaluate_all(
-            """els => els.slice(0).map(el =>
-              [...el.querySelectorAll('th')].map(e => e.textContent.trim())
-            )"""
-        )
+            "thead.govuk-table__head th[class*=browse__all__desktop__header]"
+        ).evaluate_all("""els => els.map(e => e.textContent.trim())""")
 
-        rows = aau_user_page.locator("#tbl_result tr:visible").evaluate_all(
-            """els => els.slice(1).map(el =>
-              [...el.querySelectorAll('td')].map(e => e.textContent.trim())
+        rows = aau_user_page.locator(
+            ".govuk-table__row.browse__table__all_desktop__row"
+        ).evaluate_all(
+            """els => els.map(el =>
+              [...el.querySelectorAll('td.browse__table__all_desktop')].map(e => e.textContent.trim())
             )"""
         )
 
         expected_rows = [
-            ["Mock 1 Department", "MOCK1 123", "05/03/2024", "83", "11"],
-            ["Testing A", "TSTA 1", "25/01/2024", "63", "5"],
+            ["MOCK1 Department", "MOCK1 123", "28/07/2023", "1", "1"],
+            ["Testing A", "TSTA 1", "25/01/2024", "73", "7"],
         ]
```

Comment:
the test values are validated against 'ayr' db , so please do not change the expected_rows values , as they already passed the tests based on the data in 'ayr' db

Reply:
this apply to all the test cases

Reply:
resolved on call, test cases now using ayr db